pilot/cmd/pilot-agent/status/server.go

```diff
@@ -57,18 +57,24 @@ var (
 
 // KubeAppProbers holds the information about a Kubernetes pod prober.
 // It's a map from the prober URL path to the Kubernetes Prober config.
-// For example, "/app-health/hello-world/livez" entry contains livenss prober config for
+// For example, "/app-health/hello-world/livez" entry contains liveness prober config for
 // container "hello-world".
-type KubeAppProbers map[string]*corev1.HTTPGetAction
+type KubeAppProbers map[string]*Prober
+
+// Prober represents a single container prober
+type Prober struct {
+	HTTPGet        *corev1.HTTPGetAction `json:"httpGet"`
```

Comment:
Not sure why this was changed to HTTPGet instead of HTTPGetAction. I believe this is why the probes are not working.